src/ftxui/component/container.cpp

```diff
@@ -61,6 +61,15 @@ bool Container::VerticalEvent(Event event) {
     }
   }
 
+  if (event == Event::Tab) {
+    if (selected_ != int(children_.size()) - 1) {
+      selected_++;
+      return true;
+    } else {
+        selected_ = 0;
+    }
```

Comment:
Suggestions:
~~~cpp
selected = (selected + 1) % children.size();
return true;
~~~

I think you should return true in both cases. Otherwise, it would cause issues when you have nested containers